# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/tests/crashtests/crashtests.list
# Commit: c8f5a8ec70c4
# Full Hash: c8f5a8ec70c4e4fd2aef1defe3d27fb6cd831d3b
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2020-03-06 09:47:39
# Description:
#   Bug 1620125 - Don't assume mVerticalMetrics has necessarily been initialized before DrawOneGlyph is called. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D65567
# ==============================================================================

diff -r 5b935c9b36ab -r c8f5a8ec70c4 gfx/tests/crashtests/crashtests.list
--- a/gfx/tests/crashtests/crashtests.list	Thu Mar 05 17:44:44 2020 +0000
+++ b/gfx/tests/crashtests/crashtests.list	Thu Mar 05 18:07:17 2020 +0000
@@ -189,4 +189,4 @@
 load 1566206.html
 load 1615141.html
 load 1615091.html
-
+load 1620125.html